zetaclient/orchestrator/bootstrap.go

```diff
@@ -0,0 +1,414 @@
+package orchestrator
+
+import (
+	"context"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	solrpc "github.com/gagliardetto/solana-go/rpc"
+	"github.com/pkg/errors"
+
+	"github.com/zeta-chain/zetacore/zetaclient/chains/base"
+	btcobserver "github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin/observer"
+	"github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin/rpc"
+	btcsigner "github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin/signer"
+	evmobserver "github.com/zeta-chain/zetacore/zetaclient/chains/evm/observer"
+	evmsigner "github.com/zeta-chain/zetacore/zetaclient/chains/evm/signer"
+	"github.com/zeta-chain/zetacore/zetaclient/chains/interfaces"
+	solbserver "github.com/zeta-chain/zetacore/zetaclient/chains/solana/observer"
+	"github.com/zeta-chain/zetacore/zetaclient/config"
+	zctx "github.com/zeta-chain/zetacore/zetaclient/context"
+	"github.com/zeta-chain/zetacore/zetaclient/db"
+	"github.com/zeta-chain/zetacore/zetaclient/metrics"
+)
+
+// backwards compatibility
```

Comment:
here is a bit more description about the constant
`// btcDatabaseFilename is the Bitcoin database file name now used in mainnet so we keep using it here for backward compatibility`